CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- `--display-name` flag to release command
```

Comment:
(Requesting changes just to block the PR so this comment can be seen before the merge 😅 )

@gris, I've talked to @murilovarela and we thought about adding the vendor to the tag name. This might be useful to also quickly install an app just by copying the tag name. What do you think?

Reply:
If you guys think that it is a cool idea, I'm all in!
Just out of curiosity did you guys evaluated the usage of lerna, bazel or some other monorepo tooling?
I've just noticed that having a tag referring to a specific app inside a monorepo can be misleading, specially if you are updating more than 1 app at the same time (you will have to push 2 tags pointing to the same commit).

Reply:
As I see we will have two commits with the version bump change, one for each app, and the tag will be pointing to that specific commit. 🤔

Reply:
Can you talk about the work you're doing in the CI bot, @murilovarela?

Reply:
Hum, I see. I was just wondering if it makes more sense to have the tags point to versions of the monorepo itself, like this https://github.com/vtex/faststore/blob/master/lerna.json

Reply:
In our case, @gris, it doesn't make sense to make all different apps have the same version since they don't need to be in sync to work. So we'd need to release a version of `vtex.admin-inventory` when `vtex.admin-logistics` changed, the way I saw.

We want to keep different tags so we can see, by the tag name, the app changed. I don't know if a `vtex.admin-logistics@{version}` tag with `vtex.logistics-carrier-graphql` changes would be a problem for us :)

Reply:
Nevermind, I've researched a little bit more and actually saw this process implemented with tags per package in other places. I'm very curious because I will implement a monorepo for my team in the future as well :D

Reply:
The CI bot doesn't use the toolbelt. It generates the tag by itself. And what we decide here, I will replicate in the ci-hub code.

Reply:
So we all agree that it is a good idea to add the vendor to the tag name, correct?

Reply:
I see, @murilovarela. I meant to share knowledge about what we're doing since @gris is curious about the monorepo initiative.

Anyway, could you add the vendor in the tag name here too? 😁

Reply:
All good, it makes sense to have the vendor for me.